examples/hello-openshift/README.md

```diff
@@ -1,11 +1,13 @@
 Hello, OpenShift!
 -----------------
 
-This example will serve an http response of "Hello OpenShift!" to [http://localhost:6061](http://localhost:6061).  To create the pod run:
+This example will serve an HTTP response of "Hello OpenShift!" to http://localhost:6061.
```

Comment:
I had the same doubt, even had it changed to :8080 before opening the PR. But indeed it is available on localhost:6061.